src/lib/utilities/conversion/units.ts

```diff
@@ -342,3 +343,97 @@ function makeUnit(unit: UnitOptions): Unit {
 			: { from: (si) => div(si, unit.value), to: (value) => mul(value, unit.value) }
 	};
 }
+
+export const defaults = [
+	'm', // Meter
+	'cm', // Centimeter
+	'km', // Kilometer
+	'mm', // Millimeter
+	'dm', // Decimeter
+	'dam', // Decameter
+	'lb', // Pound
+	'kg', // Kilogram
+	'US gal', // Gallon
+	'L', // Liter
+	'ft', // Foot
+	'mi', // Mile
+	'fl oz', // Fluid Ounce
+	'in' // Inch
+].map((u) => Units.get(u)!);
+
+const NameMappings = new Collection<LocaleString, Collection<string, string>>();
+
+function createNameMappings(locale: LocaleString, collection: Collection<string, string> = new Collection()) {
+	const t = getT(locale);
+	for (const unit of Units.values()) {
+		collection.set(`${unit.symbol}-${locale}`, t(unit.name).toLowerCase());
```

Comment:
What is the `-locale` for? As I mentioned, the algorithm sets a collection of `en-US` names pointing to the unit (so `Collection<Name, Symbol>`) so later languages can override them. That way, if 2 languages use the same name (e.g. `kilogram`), then the `en-US` version is overriden by the language. It's simply a deduplication system to have `en-US` fallback with the target language without duplicating.

Reply:
We discussed that it needed to be swapped to `Collection<Symbol, Name>` cause all of the units with prefixes and dimensions were replacing each other, and in doing so swapping it to <Symbol, Name>` the `en-US` fallback just gets completely replaced due to the symbols matching up, which is why I added the `locale` part to the key to prevent them from being replaced. Also, the results do get deduplicated before being sent to the user as to prevent it from showing multiple of the same unit from both the target language and `en-US´.

Reply:
That's why it should be `Collection<Name, Symbol>`, symbols are unique, but names aren't 😅